blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,128 @@
+
+Happy watching!
+
+\- Joshua
+
+{/* truncate */}
+
+## Key Release Notes/Breaking Changes
+
+There are several key changes with 10.9.0 that administrative users should be aware of, before getting into the more specific changes.
+
+* As always, **ensure you back up your Jellyfin data and configuration directories before upgrading**. With a major release, it's possible you will hit a bug and want to revert, and to do so, you will need to restore from a backup.
+
+* Ubuntu users: We have dropped support for non-LTS Ubuntu releases with 10.9.0. That is, we have not built 10.9.0 packages for any releases except 20.04 LTS, 22.04 LTS, and 24.04 LTS, and we will not publish builds for any new non-LTS releases going forward. For an explanation of why, please see [our previous blog post](https://jellyfin.org/posts/new-ci-new-repo). If you use another release, please upgrade to 24.04 LTS or switch to Docker packages.
+
+* Fedora/CentOS/RHEL/etc. users: We have dropped our official RPM packages and suggest moving to [the 3rd-party RPMFusion repository](https://admin.rpmfusion.org/pkgdb/package/free/jellyfin/). Support for RHEL-like distributions has been a major pain point for us for a very long time, and we feel that letting the community over at RPMFusion handle this is in our best interest going forward, similar to how Arch, Gentoo, etc. packages are handled. For more details, please see [our previous blog post](https://jellyfin.org/posts/new-ci-new-repo).
+
+* Docker users: We now offer GHCR as an alternative container registry in addition to DockerHub. You can pull images from the new registry via URIs like `ghcr.io/jellyfin/jellyfin:latest`. Don't worry, we have no plans to drop DockerHub as a container registry, but we feel providing both gives users more choice and flexibility.
+
+## The Big Changes
+
+This release has been nearly (over?) two years in the making, so we're really glad to finally get it out to you. The long cycle does mean quite an extensive changelog however, with well over 1100 pull requests merged into our `master` branch since 10.8.0 first dropped back in 2022. This section will detail all the big changes, broken up by general function and area of the system. In addition to these big items, there are hundreds of smaller bugfixes, quality-of-life tweaks, and other minor changes that we won't touch on here, but if you want to see the full list, you can do so over on the GitHub releases for [the core server](https://github.com/jellyfin/jellyfin/releases/tag/v10.9.0) and [the Web UI](https://github.com/jellyfin/jellyfin-web/releases/tag/v10.9.0).
+
+### General
+
+* We now support "trickplay" a.k.a. live video scrubbing. When scrubbing through a video with this enabled, you will be able to see a live preview of the video at that timestamp.
```

Comment:
Do we want to call out that on release day not all clients will support trickplay images, so users will need to watch their chosen client for updates?

Reply:
Could be worth mentioning but it's not only trick play but also all other new features on each release.

Reply:
So more of a general note / reminder that this adds the server capability, but clients will also require updates to use it.

Reply:
Added client support note.